src/contact.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="css/main.scss" rel="stylesheet" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Be+Vietnam" />
+    <title>Seaborne</title>
+    <meta name="description" content="To-do: Add description" />
+    <meta property="og:type" content="website" />
+    <meta property="og:title" content="Seaborne" />
+    <meta property="og:description" content="To-do: Add description" />
+    <meta property="og:image" content="To-do: Add share card image" />
+    <meta property="og:site_name" content="Seaborne" />
+    <meta property="og:locale" content="en_US" />
+    <meta name="twitter:title" content="Seaborne" />
+    <meta name="twitter:description" content="To-do: Add description" />
+    <meta name="twitter:image" content="To-do: Add share card image" />
+    <meta name="twitter:site" content="To-do" />
+    <meta name="twitter:creator" content="To-do" />
+    <meta name="twitter:card" content="summary_large_image" />
+    <link rel="icon" href="" />
+  </head>
```

Comment:
You sure you don't want to turn this into a partial so it can be re-used in other pages? Seems like it can be done with [posthtml-include](https://github.com/posthtml/posthtml-include).

Reply:
posthtml-include was muuuch cleaner than the way I was doing it with Parcel Bundler!! Thanks :-D